src/common/libutil/uri.c

```diff
@@ -0,0 +1,64 @@
+/************************************************************\
+ * Copyright 2021 Lawrence Livermore National Security, LLC
+ * (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+ *
+ * This file is part of the Flux resource manager framework.
+ * For details, see https://github.com/flux-framework.
+ *
+ * SPDX-License-Identifier: LGPL-3.0
+\************************************************************/
+
+#if HAVE_CONFIG_H
+#include "config.h"
+#endif
+#include <stdlib.h>
+#include <string.h>
+
+#include "src/common/libyuarel/yuarel.h"
+#include "popen2.h"
+#include "read_all.h"
+#include "log.h"
+
+static void nullify_newline (char *str)
+{
+    int n;
+    if (str && (n = strlen (str) > 0)) {
```

Comment:
Missing set of parens around `n = strlen (str)`?

Reply:
Yeah, sorry, typo that my local compiler didn't catch. Fixed now.